platform/javascript/js/engine/utils.js

```diff
@@ -1,49 +1,48 @@
-var Utils = { // eslint-disable-line no-unused-vars
+const Utils = { // eslint-disable-line no-unused-vars
 
-	createLocateRewrite: function(execName) {
+	createLocateRewrite: function (execName) {
 		function rw(path) {
 			if (path.endsWith('.worker.js')) {
-				return execName + '.worker.js';
-			} else if (path.endsWith('.audio.worklet.js')) {
-				return execName + '.audio.worklet.js';
-			} else if (path.endsWith('.js')) {
-				return execName + '.js';
-			} else if (path.endsWith('.wasm')) {
-				return execName + '.wasm';
+				return `${execName}.worker.js`;
+			} if (path.endsWith('.audio.worklet.js')) {
```

Comment:
Could also all be moved on a newline. (Again, if that's common JS codestyle, ignore this.)